.travis.yml

```diff
@@ -23,10 +23,12 @@ env:
         - DEPENDS="numpy scipy matplotlib h5py pillow"
         - PYDICOM=1
         - INSTALL_TYPE="setup"
+        - EXTRA_WHEELS="https://5cf40426d9f06eb7461d-6fe47d9331aba7cd62fc36c7196769e4.ssl.cf2.rackcdn.com"
+        - PIP_FLAGS="--find-links=$EXTRA_WHEELS"
```

Comment:
Oops - sorry - that should be EXTRA_PIP_FLAGS, used explicitly below.  I've changed it...